src/main/webapp/javascript/event.js

```diff
@@ -0,0 +1,94 @@
+// Get userId and userName for current page
+let userId = sessionStorage.getItem("userId");
+let userName = sessionStorage.getItem("userName");
+
+// Get tripId and tripName for current page
+let tripId = sessionStorage.getItem("tripId");
+let tripTitle = sessionStorage.getItem("tripTitle");
+console.log("tripId: " + tripId);
+
+var msgDiv = document.getElementById("status-msg");
+var submit = document.getElementById("submit-btn");
+
+// set header
+// Show selected user name in header
+let btnLogin = document.getElementById("header-btn")
+
+if (userName === "" || userName === null) {
```

Comment:
Ok to leave this for now.
But this is probably common for all pages, so could be moved to a common function and called on onLoad().